napari/_qt/menus/_util.py

```diff
@@ -102,6 +102,8 @@ def populate_menu(menu: QMenu, actions: List['MenuItem']):
                 def _setchecked(e, action=action):
                     action.setChecked(e.value if hasattr(e, 'value') else e)
 
+        action.setData(ax)
```

Comment:
There is a widget leak in the tests, I'm going to guess this is either here.

Reply:
Yep, is this one.